drivers/overlay/composefs_supported.go

```
@@ -75,7 +75,13 @@ func enableVerityRecursive(path string) error {
 	return filepath.WalkDir(path, walkFn)
 }
 
-func generateComposeFsBlob(toc []byte, destFile string) error {
+func getErofsBlob(dataDir string) string {
+	return filepath.Join(dataDir, "erofs.blob")
+}
+
+func generateComposeFsBlob(toc []byte, composefsDir string) error {
+	_ = os.MkdirAll(composefsDir, 0o700)
```

Any reason to ignore errors here?

---

I'll fix that